test/unit/vl-search-filter.test.html

```diff
@@ -40,6 +40,27 @@ <h2>Test</h2>
     </template>
   </test-fixture>
 
+  <test-fixture id="vl-search-filter-name-valuefixture">
+    <template>
+      <div is="vl-search-filter">
+        <form>
+          <section>
+            <h2>Test</h2>
+            <div>
+                <label is="form-label" >Naam</label>
+                <input is="vl-input-field" type="text" name="naam" value=""/>
+                <input is="vl-input-field" type="text" name="voornaam" value="Jos"/>
+                <input type="checkbox" name="kleur1" value="rood" checked /> 
+                <input type="checkbox" name="kleur2" value="blauw" /> 
+                <input type="radio" name="geslacht" value="man" checked>
+                <input type="radio" name="geslacht" value="vrouw">
+              </div>
+          </section>
+        </form>
+      </div>
+    </template>
+  </test-fixture>
+
```

Comment:
ook vl-select, vl-multiselect, vl-datepicker, vl-... testen?

Reply:
het zou kunnen dat die nog moeten uitgebreid worden om de value te kunnen opvragen.

Reply:
ik heb enkel voor native elementen gekozen nu... maar de vraag is ook of ik ze allemaal moet testen (en wat allemaal is)

Reply:
Goede vraag, op zich maakt het niet uit dat die ook getest worden als die zich aan de API houden. Dat zal nog niet overal zo zijn en moeten we dus de niet native varianten zoals `vl-checkbox` en `vl-datepicker` eerst uitbreiden met een value getter en setter.

Reply:
ok, wat doe ik dan nu?

Reply:
ik wil alleen zeker weten dat dit samenwerkt met alle native componenten en alle "form" componenten uit onze bibliotheek. als je dat kan garanderen maakt het mij niet uit waar de testen daarvoor zitten.

Reply:
Je kan hier nooit al de componenten in gaan testen. De API is duidelijk en al de input elementen die niet voldoen aan de API zullen niet werken, dat is in mijn ogen de verantwoordelijkheid voor de elementen en niet voor de `vl-ui-search-filter`.

Reply:
ok, ik check voor:
color, date, datetime-local, email, file, hidden, image, month, number, password, radio, range, reset, search, tel, text, time, url, week.
Zijn er nog die ik moet doen?

Uit onze bibliotheek wil zeggen extra dependencies... daar kan ik  deze testen die reeds een value attribuut hebben?

Reply:
Comment van Tom had ik nog niet gelezen toen ik de mijne typte.
Wat wordt het besluit?